components/suggest/src/suggest.udl

```diff
@@ -26,6 +32,7 @@ dictionary Suggestion {
     sequence<u8>? icon;
     string? impression_url;
     string? click_url;
+    ProviderType? provider;
```

Comment:
```suggestion
    ProviderType provider;
```

...And, if you do the `provider` trick I mentioned above, this can change to be non-optional!